### PR TITLE
Another fixes and additions

### PR DESCRIPTION
**Pull request by Symmachus**
Previous PR will be focused on fixing of the vanilla duplicates, while this one - on all other stuff.

### CODE REVIEW

File: WTWSMS/history/characters/roman.txt [hunk 1]
```diff
@@ -3871,9 +3871,30 @@
...
+	culture="roman"
+	420.1.1={
+		birth=yes
+	}
+	490.1.1={
+		death=yes
+	}
+}
+190119 = {
+	name="Castalius Innocentius" #PLRE vol. Castalius Innocentius Audax 3
```
Comment (IhateTrains) [inline WTWSMS/history/characters/roman.txt]:
```suggestion
	name="Castalius Innocentius" #PLRE vol. 2 Castalius Innocentius Audax 3
```

Comment (Symmachus) [reply WTWSMS/history/characters/roman.txt]: Fixed PLRE link

File: WTWSMS/history/characters/roman.txt [hunk 2]
```diff
@@ -3871,9 +3871,30 @@
 		death="530.5.2"
 	}
 }
-#190114 free, previously belonged to a duplicate of 1003501
-#190119 free, previously belonged to a duplicate of 1003500
-
+190114 = {
```
Comment (IhateTrains) [inline WTWSMS/history/characters/roman.txt]: OK.

File: WTWSMS/history/characters/roman.txt [hunk 3]
```diff
@@ -3871,9 +3871,30 @@
...
+	religion="nicene"
+	culture="roman"
+	420.1.1={
+		birth=yes
+	}
+	490.1.1={
+		death=yes
+	}
+}
+190119 = {
```
Comment (IhateTrains) [inline WTWSMS/history/characters/roman.txt]: OK.

File: WTWSMS/history/titles/e_persia.txt [hunk 4]
```diff
@@ -8,7 +8,7 @@
 	law = centralization_3
 }
 11.1.1={
-	holder=159546
+	holder=159546
```
Comment (IhateTrains) [inline WTWSMS/history/titles/e_persia.txt]: This is unneeded.

File: WTWSMS/history/characters/kurdish.txt [hunk 5]
```diff
@@ -13,7 +13,7 @@
 	}
 }
 159547 = {
-	name="Balax�" #Balash
+	name="Balaxš" #Balash
```
Comment (IhateTrains) [inline WTWSMS/history/characters/kurdish.txt]: I don't know what happened here and in two characters under this, probably encoding changed.

File: WTWSMS/history/titles/e_persia.txt (hunk 4 above)
Comment (Symmachus) [reply WTWSMS/history/titles/e_persia.txt]: > 
> 
> This is unneeded.

Hm, i don't changed this.

File: WTWSMS/history/characters/kurdish.txt (hunk 5 above)
Comment (Symmachus) [reply WTWSMS/history/characters/kurdish.txt]: Yes, encoding was changed, now i fixed it.

File: WTWSMS/history/titles/e_persia.txt (hunk 4 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/e_persia.txt]: A space has been added after the ID.

File: WTWSMS/history/characters/egyptian_arabic.txt [hunk 6]
```diff
@@ -284,7 +284,7 @@
 161509 = {
 	name="Khail" #I
 	religion="nicene"
-	culture="egyptian_arabic"
+	culture="coptic"
```
Comment (IhateTrains) [inline WTWSMS/history/characters/egyptian_arabic.txt]: All the changes in this file are OK.

File: WTWSMS/history/titles/e_persia.txt (hunk 4 above)
Comment (Symmachus) [reply WTWSMS/history/titles/e_persia.txt]: > 
> 
> A space has been added after the ID.

I hope now all fine.

File: WTWSMS/history/characters/kurdish.txt [hunk 7]
```diff
@@ -3,7 +3,7 @@
 	# Vonones II, King of Parthia in 51, King of Atropatene in 11 - 50
 	dynasty=100685
 	religion="zoroastrian"
-	culture="parthian"
+	culture="kurdish"
```
Comment (IhateTrains) [inline WTWSMS/history/characters/kurdish.txt]: Was he a Kurd?

Comment (Symmachus) [reply WTWSMS/history/characters/kurdish.txt]: > 
> 
> Was he a Kurd?

I'm used ''replace all'' option and accidentaly changed his culture.

File: WTWSMS/history/characters/kurdish.txt [hunk 8]
```diff
@@ -112,6 +112,7 @@
 	dynasty=1029098
 	religion="zoroastrian"
 	culture="kurdish"
+	trait=zurvanist_trait
```
Comment (IhateTrains) [inline WTWSMS/history/characters/kurdish.txt]: Is this historical or random?

Comment (Symmachus) [reply WTWSMS/history/characters/kurdish.txt]: > 
> 
> Is this historical or random?

This was remnant of the school traits, which i added previously.

**Comment by IhateTrains** [comment]
Please don't add new changes to this PR, I can quickly review it.

File: WTWSMS/history/characters/levantine_arabic.txt [hunk 9]
```diff
@@ -58,7 +58,7 @@
 	intrigue=5
 	stewardship=5
 	religion="nicene"
-	culture="levantine_arabic"
+	culture="aramean"
```
Comment (IhateTrains) [inline WTWSMS/history/characters/levantine_arabic.txt]: Did you use "replace all" in this file or was every character checked separately?

Comment (Symmachus) [reply WTWSMS/history/characters/levantine_arabic.txt]: > 
> 
> Did you use "replace all" in this file or was every character checked separately?

Every character was checked separately. Don't bother yourself reviewing of this file, Loup already doing it in previous PR.

Comment (IhateTrains) [reply WTWSMS/history/characters/levantine_arabic.txt]: OK, to be honest I'm relieved 😀

File: WTWSMS/history/titles/c_messina.txt [hunk 10]
```diff
@@ -28,6 +28,9 @@
 461.1.1={
 	holder=190103 # Caelius Aconius Probianus 
 }
+468.1.1={
```
Comment (IhateTrains) [inline WTWSMS/history/titles/c_messina.txt]: Shouldn't this be after `463.1.1={
	holder=190101 # Caecina Decius Basilius 
}` (for chronological order)?

Comment (Symmachus) [reply WTWSMS/history/titles/c_messina.txt]: > 
> 
> Shouldn't this be after `463.1.1={ holder=190101 # Caecina Decius Basilius }` (for chronological order)?

Ok, i will add this line.

File: WTWSMS/history/titles/k_sindh.txt [hunk 11]
```diff
@@ -15,7 +15,7 @@
 510.1.1={
 	holder=36346 # Rai Bhaimani Rais 
 }
-550.1.1={
+540.1.1={
```
Comment (IhateTrains) [inline WTWSMS/history/titles/k_sindh.txt]: OK.

File: WTWSMS/history/titles/e_axum.txt [hunk 12]
```diff
@@ -50,10 +50,10 @@
 500.1.1={
 	holder=32204 # Ousas
 }
-520.1.1={
+512.1.1={
```
Comment (IhateTrains) [inline WTWSMS/history/titles/e_axum.txt]: OK.

File: WTWSMS/history/titles/e_axum.txt [hunk 13]
```diff
@@ -50,10 +50,10 @@
...
 	holder=33188 # Kaleb
 }
-550.1.1={
+540.1.1={
```
Comment (IhateTrains) [inline WTWSMS/history/titles/e_axum.txt]: OK.

File: WTWSMS/history/titles/d_domnonia.txt [hunk 14]
```diff
@@ -5,6 +5,6 @@
 470.1.1={
 	holder = 212881 # Bleiddig Sabis
 }
-490.1.1={
+510.1.1={
```
Comment (IhateTrains) [inline WTWSMS/history/titles/d_domnonia.txt]: OK.

File: WTWSMS/history/titles/d_broerec.txt [hunk 15]
```diff
@@ -5,6 +5,6 @@
 470.1.1={
 	holder = 212892 # Meriadog Condevicnum
 }
-490.1.1={
+510.1.1={
```
Comment (IhateTrains) [inline WTWSMS/history/titles/d_broerec.txt]: OK.

File: WTWSMS/history/titles/c_somerset.txt [hunk 16]
```diff
@@ -34,6 +34,9 @@
 480.1.1={
 	holder = 2 # Geraint Llyngesic ap Erbin
 }
+514.1.1={
+	holder = 228000 # Cado ap Gerren
+}
```
Comment (IhateTrains) [inline WTWSMS/history/titles/c_somerset.txt]: (I don't know much about Briton history)
Could you explain this change?

Comment (Symmachus) [reply WTWSMS/history/titles/c_somerset.txt]: > 
> 
> (I don't know much about Briton history)
> Could you explain this change?

Previous holder died in 514, so between his death and anglo-saxon occupation i gave these counties to heir.

File: WTWSMS/history/titles/c_somerset.txt [hunk 17]
```diff
@@ -34,6 +34,9 @@
 480.1.1={
 	holder = 2 # Geraint Llyngesic ap Erbin
 }
+514.1.1={
```
Comment (IhateTrains) [inline WTWSMS/history/titles/c_somerset.txt]:
```suggestion
514.1.1={ # previous holder dies in 514, so his heir has been added to fill the 514-527 gap
```

File: WTWSMS/history/titles/c_messina.txt (hunk 10 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/c_messina.txt]: OK. Once this file is fixed (other files with him are OK) the PR can be merged.

Comment (Symmachus) [reply WTWSMS/history/titles/c_messina.txt]: > 
> 
> OK. Once this file is fixed (other files with him are OK) the PR can be merged.

Now this is fixed.

**Pull request merged**